packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -252,6 +277,21 @@ export class CdpAutomation {
 
           return this.sendDebuggerCommandFn('Network.deleteCookies', _.pick(cookieToBeCleared, 'name', 'domain'))
         })
+
+      case 'clear:cookies':
+        return Bluebird.mapSeries(data as CyCookieFilter[], async (cookie) => {
+          // resolve with the value of the removed cookie
+          // also, getting the cookie via CDP first will ensure that we send a cookie `domain` to CDP
+          // that matches the cookie domain that is really stored
```

Comment:
the amount of CDP requests we send for cookies stuff is crazy, sometimes the CDP debug logs scroll for minutes on the cookies e2e tests